dir.props

```diff
@@ -8,6 +8,12 @@
     <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
   </PropertyGroup>
 
+  <!-- Version numbers for both managed & native binaries -->
+  <PropertyGroup>
```

Comment:
hmmm testbuild seems to be polluting the product binaries folder. https://github.com/dotnet/coreclr/blob/master/build-test.cmd#L107
This was added by @sejongoh